database/Other/devtools.json

```diff
@@ -22,7 +22,7 @@
     },
     {
         "name": "Swagger",
-        "description": "This site will help you to create API docementaion for your APIs",
+        "description": "This site will help you to create docementaion for your APIs",
```

Comment:
what's decementaion?

Reply:
"decementaion" spelling error
sorry for that 
I have updated it to "This site will help you to create documentation for your APIs."